templates/CSD-1.html.tmpl

```diff
@@ -236,200 +338,405 @@
-                    <th>Requirement</th>
-                    <th>Applicability</th>
-                </tr>
-                {{#each data.requirements.proceduralconstraints}}
-                <tr>
-                    <td class="lean" rowspan="3">{{key}}</td>
-                    <td class="lean">URS</td>
-                    <td class="content-wrappable">Summary: {{{ursName}}}{{#if ursDescription}}<br>{{{ursDescription}}}{{/if}}</td>
-                    <td class="lean">{{applicability}}</td>
-                </tr>
-                <tr>
-                    <td class="lean">FS</td>
-                    <td class="content-wrappable">Summary: {{{fsName}}}{{#if }}<br>{{{fsDescription}}}{{/if}}</td>
-                    <td class="lean">{{applicability}}</td>
-                </tr>
-                <tr>
-                    <td class="lean">CS</td>
-                    <td class="content-wrappable">N/A</td>
-                    <td class="lean">N/A</td>
-                </tr>
+            {{#if data.requirements.proceduralconstraints}}
+                {{#if data.requirements.proceduralconstraints.noepics}}
+                <table>
+                    <tr>
+                        <th>No.</th>
+                        <th>Document</th>
+                        <th>Requirement</th>
+                        <th>Applicability</th>
+                    </tr>
+                    {{#each data.requirements.proceduralconstraints.noepics}}
+                    <tr>
+                        <td class="lean" rowspan="3">{{key}}</td>
+                        <td class="lean">URS</td>
+                        <td class="content-wrappable">Summary: {{{ursName}}}{{#if ursDescription}}<br>{{{ursDescription}}}{{/if}}</td>
+                        <td class="lean">{{applicability}}</td>
+                    </tr>
+                    <tr>
+                        <td class="lean">FS</td>
+                        <td class="content-wrappable">Summary: {{{fsName}}}{{#if }}<br>{{{fsDescription}}}{{/if}}</td>
+                        <td class="lean">{{applicability}}</td>
+                    </tr>
+                    <tr>
+                        <td class="lean">CS</td>
+                        <td class="content-wrappable">N/A</td>
+                        <td class="lean">N/A</td>
+                    </tr>
+                    {{/each}}
+                </table>
+                {{/if}}      
+
+                {{#each data.requirements.proceduralconstraints.epics}}
+                <h4 id="section_4_2_{{epicIndex}}"><span>4.2.{{epicIndex}}</span>{{epicName}}</h4>
+                <span>Content for the epic: </span>
+                <table>
+                    <tr>
+                        <th>No.</th>
+                        <th>Document</th>
+                        <th>Requirement</th>
+                        <th>Applicability</th>
+                    </tr>
+                    {{#each data.requirements.proceduralconstraints.epics}}
```

Comment:
I'm not sure about this line. The Each seems to be duplicated with the one some lines above. Please could you provide @metmajer @jorge-romero an example document to see how this is shown?

Reply:
This is for the Stories related with epics, the first block in the code provides the stories not related with epics and the second block to Stories grouped by epics

Reply:
I'll send you by email an example

Reply:
I was refering to this block
`
                {{#each data.requirements.functionalrequirements.epics}}
                    <h5 id="section_3_2_1_{{epicIndex}}"><span>3.2.1.{{epicIndex}}</span>{{epicName}}</h5>
                    <span>Content for the epic: </span>
                <table>
                    <tr>
                        <th>No.</th>
                        <th>Document</th>
                        <th>Requirement</th>
                        <th>Applicability</th>
                    </tr>
                    {{#each data.requirements.functionalrequirements.epics}}
`

{{#each data.requirements.functionalrequirements.epics}} is repeated twice